client/internal/dns/mock_test.go

```diff
@@ -23,3 +23,16 @@ func (rw *mockResponseWriter) Close() error              { return nil }
 func (rw *mockResponseWriter) TsigStatus() error         { return nil }
 func (rw *mockResponseWriter) TsigTimersOnly(bool)       {}
 func (rw *mockResponseWriter) Hijack()                   {}
+
+type mockHostManager struct {
```

Comment:
we got the `mockHostConfigurator` in the host.go that might be removed. There is a `newNoopHostMocker` that needs update as well.

Reply:
Fixed